tools/pot/openvino/tools/pot/graph/graph_utils.py

```diff
@@ -34,6 +34,7 @@ def load_graph(model_config, target_device='ANY'):
         apply_pot_transformations(model, target_device.encode('utf-8'))
         bin_path = serialized_bin_path
         xml_path = serialized_xml_path
+        # TODO: replace by openvino.runtime.serialize
```

Comment:
Why cannot be changed in this PR?

Reply:
Because POT CI uses openvino from the specific package, not from source branch